contracts/AxelarGateway.sol

```diff
@@ -50,13 +51,26 @@ contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
     // solhint-disable-next-line var-name-mixedcase
     address internal immutable AUTH_MODULE;
     // solhint-disable-next-line var-name-mixedcase
+    address internal immutable GOVERNANCE_MODULE;
+    // solhint-disable-next-line var-name-mixedcase
+    address internal immutable MINT_LIMIT_MODULE;
+    // solhint-disable-next-line var-name-mixedcase
     address internal immutable TOKEN_DEPLOYER_IMPLEMENTATION;
 
-    constructor(address authModule_, address tokenDeployerImplementation_) {
+    constructor(
+        address authModule_,
+        address governanceModule_,
+        address mintLimitModule_,
+        address tokenDeployerImplementation_
+    ) {
         if (authModule_.code.length == 0) revert InvalidAuthModule();
+        if (governanceModule_.code.length == 0) revert InvalidGovernanceModule();
+        if (mintLimitModule_.code.length == 0) revert InvalidGovernanceModule();
```

Comment:
```suggestion
        if (mintLimitModule_.code.length == 0) revert InvalidMintLimitModule();
```
Or use a more generic error for all of them such as `InvalidContractAddress`/`NotAContract`